docs/src/GettingStarted/RunningClimateMachine.md

```diff
@@ -28,27 +28,42 @@ $ julia --project experiments/AtmosGCM/heldsuarez.jl
 ## Input and output
 
 The `ClimateMachine` provides the [`ArtifactWrappers`](@ref
-ArtifactWrappers-docs) module to assist a driver in sourcing input data
-for a simulation, but any mechanism may be used.
+ArtifactWrappers-docs) module to assist a driver in sourcing input data for a
+simulation, but any mechanism may be used.
 
 Output takes the form of various [groups of diagnostic variables](@ref
-Diagnostics) that are written to NetCDF files at specified intervals
-by the `ClimateMachine` when configured to do so by a driver. A
-driver may alternatively (or additionally) configure output of the
-conservative and auxiliary state variables to VTK files. However,
-the default behavior of the `ClimateMachine` is to not generate
-any output, irrespective of driver setup. The user must explicitly
-enable output by using the appropriate [command line argument](@ref
-ClimateMachine-args) (e.g. `--diagnostics default`) or by setting
-the appropriate [environment variable](@ref ClimateMachine-envvars)
-(e.g. `CLIMATEMACHINE_SETTINGS_DIAGNOSTICS=default`).
-
-## [Command line arguments](@id ClimateMachine-args)
-
-The `ClimateMachine` allows some aspects of its behavior to be controlled
-by command line arguments. Many drivers make use of this facility
-(but some do not). The command line options recognized can be examined
-interactively, for example:
+Diagnostics-vars) that are written to NetCDF files at specified intervals by
+the `ClimateMachine` when configured to do so by a driver.
```

Comment:
```suggestion
Diagnostics-vars) that are written to NetCDF files at user-specified intervals
in the `ClimateMachine` driver `Callbacks`.
```
Maybe include a ref to the callbacks / driver?

Reply:
The suggestion is just a suggestion. I'm not sure if what I wrote is more transparent, but I think something along those lines would be clear/helpful